### ISSUE TITLE
Invalid type for path "rollbar.exception_sample_rates". Expected scalar, but got array.

### ISSUE THREAD

**Issue by JoydS** [rollbar/rollbar-php-symfony3-bundle#25]
Hello,
It's impossible for us to use exception_sample_rates configuration.
Here the error message we have : 

>     Invalid type for path "rollbar.exception_sample_rates". Expected scalar, but got array.

Here the configuration :

>    exception_sample_rates:
>        - \Symfony\Component\Security\Core\Exception\AccessDeniedException: 0
>        - \Symfony\Component\HttpKernel\Exception\NotFoundHttpException: 0
>        - \Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: 0
>        - \Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: 0

**Comment by rivkahstandig3636** [comment]
Thanks for reporting this! We'll take a look.

**Comment by JoydS** [comment]
Any update about this issue ? Cause I can't use this bundle without this :(

**Comment by rivkahstandig3636** [comment]
@ArturMoczulski Can you please take a look at this?

**Comment by ArturMoczulski** [comment]
Yes. This needs to get fixed. I'm adding this to the next release.

**Comment by JoydS** [comment]
Hi guys, do you have any update on this issue ? 
Thank you !

**Comment by danielfuerst** [comment]
Could you provide a sample configuration?
The following does not seem to work:
```    exception_sample_rates:
       \Symfony\Component\Security\Core\Exception\AccessDeniedException: 0
       \Symfony\Component\HttpKernel\Exception\NotFoundHttpException: 0
       \Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: 0
       \Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: 0```
